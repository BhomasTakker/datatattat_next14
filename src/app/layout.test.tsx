import "@testing-library/jest-dom";
import { render, screen } from "@testing-library/react";
import RootLayout from "./layout";

describe("RootLayout", () => {
	it("renders children correctly", async () => {
		render(await RootLayout({ children: <span>Test Child</span> }));
		expect(screen.getByText("Test Child")).toBeInTheDocument();
	});

	describe("Snaps", () => {
		it("renders Layout Unchanged", async () => {
			const { container } = render(
				await RootLayout({ children: <span>Test Child</span> })
			);
			expect(container).toMatchSnapshot();
		});
	});
});
