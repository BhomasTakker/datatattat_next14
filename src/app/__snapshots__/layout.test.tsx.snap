// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RootLayout Snaps renders Layout Unchanged 1`] = `
<div>
  <div>
    <span>
      Test Child
    </span>
  </div>
</div>
`;
